### PR TITLE
updates per latest tbdex spec changes

### PR DESCRIPTION
**Pull request by jiyoonie9**
- pull in latest schemas
- add `payoutUnitsPerPayinUnit` to quote
- `amount` > `subtotal`, and net new field `total`

### CODE REVIEW

File: tbdex/validator/json-schemas/quote.schema.json
Comment (mistermoe) [inline tbdex/validator/json-schemas/quote.schema.json:1]: @jiyoontbd we'd want to make this change in the `tbdex` repo itself. This schema is copied from the submodule in this repo

Comment (jiyoonie9) [reply tbdex/validator/json-schemas/quote.schema.json:1]: got it - will make change to the tbdex repo as part of a jumbo pr i'm about to start

related q: why do we copy from submodule in this repo? are we not able to use the submodule files directly?

Comment (mistermoe) [reply tbdex/validator/json-schemas/quote.schema.json:1]: Correct! Go modules don't have submodule support

Comment (jiyoonie9) [reply tbdex/validator/json-schemas/quote.schema.json:1]: done

File: tbdex/quote/quote.go [hunk 1]
```diff
@@ -223,17 +231,24 @@ func DetailsInstruction(p *PaymentInstruction) QuoteDetailsOption {
...
+func NewQuoteDetails(currencyCode string, subtotal decimal.Decimal, opts ...QuoteDetailsOption) QuoteDetails {
 	q := quoteDetailsOptions{}
 	for _, opt := range opts {
 		opt(&q)
 	}
+
+	total := subtotal
+	if !q.Fee.IsZero() {
+		total = subtotal.Add(q.Fee)
+	}
```
Comment (tomdaffurn) [inline tbdex/quote/quote.go]: Could do this. It works out the same whether Fee is zero or not
```suggestion
	total := subtotal.Add(q.Fee)
```

**Review by tomdaffurn — APPROVED** [review]

File: tbdex/quote/quote.go (hunk 1 above)
Comment (jiyoonie9) [reply tbdex/quote/quote.go]: tested and done! thanks https://github.com/TBD54566975/tbdex-go/pull/53/commits/173ddb057b2c2f6d8ba21216b79fa3c67fc52116

**Pull request merged**